helpers/runtime.rb

```diff
@@ -9,4 +9,23 @@ def get_runtime_jwt_payload
     rescue JWT::DecodeError
     end
   end
+
+  def get_scope_from_github(runner, run_id)
+    return unless run_id
```

Comment:
Nit: We can emit log here to check any further issues more easily.

Reply:
Since we log all request URLs and include the runId in the path query, we have these logs at some point.

Reply:
I added it